terraso_backend/apps/collaboration/graphql/memberships.py

```diff
@@ -0,0 +1,88 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+import django_filters
+import graphene
+import structlog
+from graphene import relay
+from graphene_django import DjangoObjectType
+
+from apps.graphql.schema.commons import TerrasoConnection
+
+from ..models import Membership, MembershipList
+
+logger = structlog.get_logger(__name__)
+
+
+class CollaborationMembershipListNode(DjangoObjectType):
+    id = graphene.ID(source="pk", required=True)
+    account_membership = graphene.Field("apps.collaboration.graphql.CollaborationMembershipNode")
```

Comment:
You use the string when you wnat to avoid ciruclar dependencies, I think at some point in the implementation I had that issue, I will check if that is still the case, maybe I can use it directly